src/feature-flags/featureFlags.tsx

```diff
@@ -12,6 +13,7 @@ export const FeatureFlagsProvider: React.FC = ({ children }) => {
     <FlagsProvider
       flags={{
         useRt: process.env.GATSBY_USE_RT === "true",
+        showRateOfSpreadTab: true,
```

Comment:
Let's add a TODO on this to remove the feature flag once #202 has been completed.